CONTRIBUTING.md

```diff
@@ -31,6 +31,7 @@ into three categories:
 3. Comment on the issue saying you are going to work on it.
 4. Get familar with the developer guide relevant for you:
     * For C++ developers it is available here [DEVELOPER_GUIDE.md](wiki/cpp/DEVELOPER_GUIDE.md)
+    * For Python developers, cuML Python generally follows scikit-learn guidelines, but there are some additional considerations to ensure estimators work smoothly with all input types and configurable output types. See the documentation in `cuml.common.Base` for full details.
```

Comment:
Should we link the Python developer guide here as well? (in wiki/python/DEVELOPER_GUIDE)